src/Components/Onboarding/Wizard.tsx

```diff
@@ -63,18 +66,31 @@ class Wizard extends React.Component<Props, State> {
     const { stepComponents, redirectTo } = this.props
 
     if (currentStep >= stepComponents.length - 1) {
-      window.location.href = redirectTo
+      this.setState({
+        percentComplete: 1,
+      })
+      setTimeout(() => {
+        window.location.href = redirectTo
+      }, 500)
```

Comment:
This lets you see 100% completion before redirecting the page. I found it didn't feel as satisfying without this 😛